services/sonar/sonar.legacyservice.js

```diff
@@ -0,0 +1,267 @@
+// 'use strict'
+
```

Comment:
Kept this file around for easier side-by-side comparisons in my local env, but will be deleting it (and the tester) shortly